### PR TITLE
Adding Chris' bio and pic

### PR DESCRIPTION
**Pull request by castleman**

### CODE REVIEW

**Comment by davetron5000** [comment]
woot! :+1:

**Comment by nickflux** [comment]
:+1:

**Comment by jstrait** [comment]
:boom:

**Comment by yorthehunter** [comment]
:horse: :beers: looks good to me! :minibus: :japanese_ogre:

File: _data/teams.yml [hunk 1]
```diff
@@ -262,6 +262,22 @@
       links:
       # links: accepts a list of "type" (e.g. "blog") and "url". Omit entirely if not using.
 
+    - name:        Chris
+      lastname:    Castleman
+      location:    San Francisco
+      division:    Client Experience
+      description: Chris is generalist with a passion for databases, CSS, and everything in between. He's also an endurance junkie who can often be found running, biking, or swimming down the Pacific Coast. He grew up in Philadelphia and then spent 10 snowy years in Buffalo before heading west to sunny San Francisco.
```
Comment (davidamcclain) [inline _data/teams.yml]: Typo. You put "**sunny** San Francisco". I assume you meant to omit the "sunny" part.

Comment (nickflux) [inline _data/teams.yml]: He moved here from Buffalo, not San Diego.

Comment (castleman) [inline _data/teams.yml]: You've clearly never been to Buffalo.

**Pull request merged**